### PR TITLE
fix csv name and OwnNamespace InstallModeType

### PR DESCRIPTION
**Pull request by jzding**
Operator-sdk requires base csv name and operator package name to be the same for `make bundle` to work.
Update csv name to hw-event-proxy-operator to be consistent with the package name. When working correctly, `make bundle` synced the InstallModeType values for the bundle and fixed the issue for OwnNamespace incorrectly set to false.

Updated operator-sdk version to v1.22.0-ocp as documented here:
https://docs.openshift.com/container-platform/4.11/cli_reference/osdk/cli-osdk-install.html

Signed-off-by: Jack Ding <jackding@gmail.com>

### CODE REVIEW

File: bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -19,22 +19,95 @@ metadata:
...
+    metadata:
+      name: redfish-basic-auth
+    type: Opaque
+    stringData:
+      username: <bmc_username>
+      password: <bmc_password>
+      # BMC host DNS or IP address
+      hostaddr: <bmc_host_ip_address>
+    ```
+  displayName: Bare Metal Event Relay Operator
```
Comment (nishant-parekh) [inline bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml:107]: do we want to use the `Bare Metal Event Relay` displayName upstream?.. we have been using `Hardware Event Proxy Operator` here so far..

File: bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -295,26 +370,24 @@ spec:
...
+  - Hardware
+  - event
+  - cloudevents
   links:
-  - name: Hw Event Proxy Operator
-    url: https://hw-event-proxy-operator.domain
-  maintainers:
-  - email: your@email.com
-    name: Maintainer Name
+  - name: Bare Metal Event Relay Operator
```
Comment (nishant-parekh) [inline bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml:386]: same as above

File: config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -28,7 +28,7 @@ metadata:
     provider: Red Hat
     repository: https://github.com/openshift/hw-event-proxy
     support: Red Hat
-  name: bare-metal-event-relay-operator.v0.0.0
+  name: hw-event-proxy-operator.v0.0.0
```
Comment (nishant-parekh) [inline config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml:31]: if we are going to use the d/s name we might want to change this back to `bare-metal-event-relay` else this is good

**Review by nishant-parekh — CHANGES_REQUESTED** [review]

File: config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (jzding) [reply config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml:31]: this was the change that broke the `make bundle`. I have to change back to use upstream name here to be consistent with the package name.

File: bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (jzding) [reply bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml]: updated

File: bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (jzding) [reply bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml]: updated

**Review by nishant-parekh — APPROVED** [review]

**Review by aneeshkp — APPROVED** [review]

File: config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (yliu127) [reply config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml:31]: Is it possible to change the other place instead? We need the user visible package name and operator name to be bare-metal-xxx. Ignore my comment if this does not have impact on the operator name we use in subs and seen in csv.

Comment (nishant-parekh) [reply config/manifests/bases/hw-event-proxy-operator.clusterserviceversion.yaml:31]: @yliu127 this is replaced in the d/s automation to bare-metal-event-relay we can leave it as hw-event-proxy-operator here.

**Pull request merged**